velox/connectors/fuzzer/FuzzerConnector.cpp

```diff
@@ -22,8 +22,11 @@ namespace facebook::velox::connector::fuzzer {
 FuzzerDataSource::FuzzerDataSource(
     const std::shared_ptr<const RowType>& outputType,
     const std::shared_ptr<connector::ConnectorTableHandle>& tableHandle,
-    velox::memory::MemoryPool* FOLLY_NONNULL pool)
-    : outputType_(outputType), pool_(pool) {
+    velox::memory::MemoryPool* FOLLY_NONNULL pool,
+    const std::shared_ptr<dwio::common::ReaderOptions>& options)
+    : outputType_(outputType), pool_(pool), options_(options) {
+  if (!options_)
```

Comment:
Not sure but the interface change required passing ReaderOptions into the class.

Reply:
Can you post the compile error without passing ReaderOptions into the class?
Also `if(!options_)` is probably redundant since this is in a constructor.

Reply:
I don't see why FuzzerDataSource needs this. I see that `FuzzeConnector::createDataSource` does.